CRVResponse/fcl/prolog_v11.fcl

```diff
@@ -9,7 +9,9 @@
 #include "Offline/CommonMC/fcl/prolog.fcl"
 BEGIN_PROLOG
 
-    DigitizationStart: 400.0 //ns after event window start (i.e. 400ns after first clock tick after POT) --> 400ns...425ns after POT
+    DigitizationStart :  200.0 //ns after event window start (400ns...425ns after POT)
```

Comment:
In the tracker we define digitizationStart/end in fcl as the effective time you want referenced to nominal proton arrival time, and then subtract the timeFromProtonsToDRMarker to get the actual value used by the digi making / hardware. Might make it easier to keep in sync if timeFromProtonsToDRMarker changes